test/e2e/README.md

```diff
@@ -30,6 +30,28 @@ Multiple testnets can be run with the `run-multiple.sh` script:
 ./run-multiple.sh networks/generated/gen-group3-*.toml
 ```
 
+Testnets running different versions of Tendermint can be generated by the
+generator. For example:
+
+```sh
+# Generate testnets randomly choosing between v0.34.21 (with a random weighting
+# of 1) and v0.34.22 (with a random weighting of 2).
+./build/generator -m "v0.34.21:1,v0.34.22:2" -d networks/generated/
+
+# "HEAD" refers to the code on the current branch.
```

Comment:
```suggestion
# "HEAD" refers to the code on the current branch (or current commit if in detached mode).
```

Reply:
I'm going to change this identifier to `local` instead for clarity and I'll update the readme accordingly.